webapp/javascript/index.jsx

```diff
@@ -45,36 +46,47 @@ function App() {
 
   return (
     <div className="app">
-      <Sidebar />
       <Switch>
-        <Route exact path="/">
-          <ContinuousSingleView />
-        </Route>
-        <Route path="/comparison">
-          <ContinuousComparisonView />
-        </Route>
-        <Route path="/comparison-diff">
-          <ContinuousDiffView />
-        </Route>
-        <Route path="/settings">
-          <Settings />
+        <Route exact path="/login">
+          <SignInPage />
         </Route>
-        <Route path="/service-discovery">
-          <ServiceDiscoveryApp />
+        <Route exact path="/signup">
+          <SignUpPage />
         </Route>
```

Comment:
Don't we have to add a condition here, since the pyroscope instance may not allow signup?